src/v2.rs

```diff
@@ -430,21 +593,54 @@ impl EszipV2 {
       }
     }
 
+    // add npm snapshot to header
+    if let Some(npm_snapshot) = self.npm_snapshot {
```

Comment:
What happens if you don't add an `npm_snapshot`? It looks like the section is omitted, which seems wrong? Shouldn't it insert an empty section instead?

Reply:
I just looked into it and it still inserts it below on line 638. I'll improve the comment